src/System.IO.Packaging/src/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10-beta-23008",
```

Comment:
You no longer need to do this since I've updated the DNX version.  You should be able to switch back to using *s.